src/service/navigation.js

```diff
@@ -624,16 +624,18 @@ export class Navigation {
     // confusing behavior e.g. when pressing "tab" button.
     // @see https://humanwhocodes.com/blog/2013/01/15/fixing-skip-to-content-links/
     // @see https://github.com/ampproject/amphtml/issues/18671
-    if (Services.platformFor(this.ampdoc.win).isIe()) {
-      const id = toLocation.hash.substring(1);
-      const elementWithId = this.ampdoc.getElementById(id);
-      if (elementWithId) {
-        if (
-          !/^(?:a|select|input|button|textarea)$/i.test(elementWithId.tagName)
-        ) {
-          elementWithId.tabIndex = -1;
+    if (!IS_ESM) {
```

Comment:
Can we inline this into `isIe`? That way every call site can just be fixed?

Reply:
Originally that's what I did, but there are a lot more callers of `isIe` and was too timid to make such a large change without the tests running fully on `esm` yet.

Reply:
Resolved with #d61b798.